validator_client/slashing_protection/Makefile

```diff
@@ -4,22 +4,29 @@ OUTPUT_DIR := interchange-tests
 TARBALL := $(OUTPUT_DIR)-$(TESTS_TAG).tar.gz
 ARCHIVE_URL := https://github.com/eth2-clients/slashing-protection-interchange-tests/tarball/$(TESTS_TAG)
 
+ifeq ($(OS),Windows_NT)
+	RMFILE:=del /F /Q
+	RMDIR:=rmdir /Q /S
+else
+	RMFILE:=rm -f
+	RMDIR:=rm -rf
+endif
+
 $(OUTPUT_DIR): $(TARBALL)
-	rm -rf $@
```

Comment:
I think this line should stay?

Reply:
Windows `del` command will fail if the file doesn't exist and there's no flag to change that behavior. So this will always fail on windows if that's left in. I tried a couple ways to work around it (using windows-shell specific "IF EXIST DIR (del DIR)") but ti wasn't working.

Reply:
yeah I had to figure out how to work around the command failing in windows if the file didn't exist.